doc/source/roadmap-detailed.rst

```diff
@@ -355,15 +355,28 @@ functions, and spheroidal wave functions. Three possible ways to handle this:
 stats
 `````
 
-This module is in good shape overall.  New functionality that's similar to
-what's already present can continue to be added; more advanced statistical
-routines may fit better in ``statsmodels``.  Some ideas for new contributions
-are:
-
-- Implementing (well-known) distributions to the ``stats.distributions``
-  framework is always welcome.
-- Continuing work on making the function signatures of ``stats`` and
-  ``stats.mstats`` more consistent, and adding tests to ensure that that
+The ``scipy.stats`` subpackage does not seek to duplicate the advanced 
```

Comment:
can you try to rephrase this. starting with what we _don't_ seek is a bit odd. Better to say what the scope _is_. And then say that more advanced functionality built on top of `scipy.stats` can be found in those other packages.